web/src/features/panels/LeftPanel.tsx

```diff
@@ -121,7 +121,7 @@ export default function LeftPanel() {
       <Routes>
         <Route path="/" element={<HandleLegacyRoutes />} />
         <Route
-          path="/zone/:zoneId"
+          path="/zone/:zoneId/*"
```

Comment:
instead of a wildcard I think we can define `production` and `emissions` as valid paths instead. It should be safer going forward. (if we decide to include this now)